package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "stoplight-elements-apis",
+  "description": "Basic React app with embedded Stoplight Elements: StoplightAPI and Stoplight Project",
+  "author": {
+    "name": "Michael Miaskowski",
+    "email": "mmiask92@gmail.com"
+  },
+  "version": "0.2.0",
+  "private": true,
```

Comment:
Right, it doesn't refer to that. Regarding publishing that's what I meant with the second phrase - that it might make sense to have it. But we're not using it in another examples and we rather won't do that accidentally.